wordclock/esp_glue.c

```diff
@@ -65,60 +69,19 @@ uint32_t Sleep(uint32_t ms) {
 	}
     return ms;
 }
-
-//function is not used any more
-void HexDump (char *desc, void *addr, int len) {
-    int i;
-    unsigned char buff[17];
-    unsigned char *pc = (unsigned char*)addr;
-
-    // Output description if given.
-    if (desc != NULL)
-        printf ("%s:\n", desc);
-
-    if (len == 0) {
-        printf("  ZERO LENGTH\n");
-        return;
-    }
-    if (len < 0) {
-        printf("  NEGATIVE LENGTH: %i\n",len);
-        return;
-    }
-
-    // Process every byte in the data.
-    for (i = 0; i < len; i++) {
-        // Multiple of 16 means new line (with line offset).
-
-        if ((i % 16) == 0) {
-            // Just don't print ASCII for the zeroth line.
-            if (i != 0)
-                printf ("  %s\n", buff);
-
-            // Output the offset.
-            printf ("  %04x ", i);
-        }
-
-        // Now the hex code for the specific character.
-        printf (" %02x", pc[i]);
-
-		// And store a printable ASCII character for later.
-		if ((pc[i] < 0x20) || (pc[i] > 0x7e))
-			buff[i % 16] = '.';
-		else
-			buff[i % 16] = pc[i];
-		buff[(i % 16) + 1] = '\0';
-	}
-
-	// Pad out last line if not exactly 16 characters.
-	while ((i % 16) != 0) {
-        printf ("   ");
-        i++;
-    }
-
-    // And print the final ASCII bit.
-    printf ("  %s\n", buff);
+void test_sysparam() {
```

Comment:
Deze functie mage er toch wel uit?

Reply:
Ja mag weg, maar eerlijk gezegt verwacht ik dat als je een update doet van de oude FW naar de nieuwe dat we dan onze settings niet meer kunnen opslaan dus dat moeten we even testen

Reply:
Ik heb even een test gedaan via jou tftp server, maar als ik van de oude versie update naar een nieuwe versie dan werkt WiFi inderdaad maar het wegschrijven van onze settings werkt dan niet meer.

En ik verwacht dat dat er mee te maken heeft dat sysparam niet meer werkt(daar was die test functie voor)

Alleen ik kan dit niet testen omdat de oude FW altijd via jou update (ik kan nu geen nieuwe FW op jou tftp server zetten)